content/blog/eng/operators-overview.md

```diff
@@ -0,0 +1,140 @@
+---
+authors: ["AndreaRR18"]
+date: "2020-03-24"
+draft: true
+share: true
+categories: [English, Operators]
+title: "Operators overview"
+type: "post"
+languageCode: "en-EN"
+toc: true
+---
+
+# Operators overview
+Operators are useful constructors and they are present in most programming languages. They are fundamental for much operations and in this article we will make an overview to better understand their properties.
+This article is the first of two articles about operators. In this article we will do an overview of the main operators' properties and in the second we will discuss about some custom operators that we use in our production code.
+
+# Definition
+An operator is a special symbol or expression that is used to check, change or combine values. Most programming languages support their construct and we can considered them like a functions but with a different syntax and semantic.
```

Comment:
You could emphasize the definition...maybe make it bold or separate from the rest of text.